### DISCUSSION TITLE
Congratulations

### DISCUSSION THREAD

**Discussion by toupswork** [linkdotnet/Blog#328] [General]
I take it you saw Nick Chapsas mentioned you and your list.add performance test on his show today?

**Comment by toupswork** [comment]
I wonder why it's slower on your processor. I'm sure it's a RISC processor..Were you able to compare the JIT compiled assembly code?

**Reply by linkdotnet** [reply]
Lol now that you mentioned it I checked it out - interesting how quickly folks picked that up.

Yes, according to the ticket it has to do with arm64 specifically for Mac. One assembler/jit instruction changed and it seems to be related to staled data. Really interesting to see.
